Error building example i2c_fujitsu_fram

**Describe the bug**
I'm trying to build one of the examples provided along with the Zephyr code base.  I haven't been able to find the reason for the resulting error. 

**To Reproduce**
Steps to reproduce the behavior:
1. with the expectation that zephyr was setup as `west init ~/zephyrproject && cd ~/zephyrproject && west update`
2. `west build -b  nrf52840dk_nrf52840 -p auto  zephyr/samples/drivers/i2c_fujitsu_fram/`
3. See error

**Expected behavior**
I am expecting the example project to build successfully

**Impact**
It's a show stopper.  I'm not sure how to use this example code for i2c.

**Logs and console output**
```
$ west build -b  nrf52840dk_nrf52840 -p auto  zephyr/samples/drivers/i2c_fujitsu_fram/
[12/100] Building C object CMakeFiles/app.dir/src/main.c.obj
FAILED: CMakeFiles/app.dir/src/main.c.obj 
ccache /Users/joshfischer/gnuarmemb/bin/arm-none-eabi-gcc -DBUILD_VERSION=zephyr-v2.6.0-1334-g57690f566e80 -DKERNEL -DNRF52840_XXAA -D_FORTIFY_SOURCE=2 -D__PROGRAM_START -D__ZEPHYR__=1 -I/Users/joshfischer/zephyrproject/zephyr/include -I/Users/joshfischer/zephyrproject/build/zephyr/include/generated -I/Users/joshfischer/zephyrproject/zephyr/soc/arm/nordic_nrf/nrf52 -I/Users/joshfischer/zephyrproject/modules/hal/cmsis/CMSIS/Core/Include -I/Users/joshfischer/zephyrproject/modules/hal/nordic/nrfx -I/Users/joshfischer/zephyrproject/modules/hal/nordic/nrfx/drivers/include -I/Users/joshfischer/zephyrproject/modules/hal/nordic/nrfx/mdk -I/Users/joshfischer/zephyrproject/zephyr/modules/hal_nordic/nrfx/. -I/Users/joshfischer/zephyrproject/modules/debug/segger/SEGGER -I/Users/joshfischer/zephyrproject/modules/debug/segger/Config -I/Users/joshfischer/zephyrproject/zephyr/modules/segger/. -isystem /Users/joshfischer/zephyrproject/zephyr/lib/libc/minimal/include -isystem /Users/joshfischer/gnuarmemb/bin/../lib/gcc/arm-none-eabi/10.2.1/include -isystem /Users/joshfischer/gnuarmemb/bin/../lib/gcc/arm-none-eabi/10.2.1/include-fixed -Os -imacros /Users/joshfischer/zephyrproject/build/zephyr/include/generated/autoconf.h -ffreestanding -fno-common -g -gdwarf-4 -fdiagnostics-color=always -mcpu=cortex-m4 -mthumb -mabi=aapcs -imacros /Users/joshfischer/zephyrproject/zephyr/include/toolchain/zephyr_stdint.h -Wall -Wformat -Wformat-security -Wno-format-zero-length -Wno-main -Wno-pointer-sign -Wpointer-arith -Wexpansion-to-defined -Wno-address-of-packed-member -Wno-unused-but-set-variable -Werror=implicit-int -fno-asynchronous-unwind-tables -fno-pie -fno-pic -fno-strict-overflow -fno-reorder-functions -fno-defer-pop -fmacro-prefix-map=/Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram=CMAKE_SOURCE_DIR -fmacro-prefix-map=/Users/joshfischer/zephyrproject/zephyr=ZEPHYR_BASE -fmacro-prefix-map=/Users/joshfischer/zephyrproject=WEST_TOPDIR -ffunction-sections -fdata-sections -std=c99 -nostdinc -MD -MT CMakeFiles/app.dir/src/main.c.obj -MF CMakeFiles/app.dir/src/main.c.obj.d -o CMakeFiles/app.dir/src/main.c.obj -c /Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c
In file included from /Users/joshfischer/zephyrproject/zephyr/include/arch/arm/aarch32/arch.h:20,
                 from /Users/joshfischer/zephyrproject/zephyr/include/arch/cpu.h:19,
                 from /Users/joshfischer/zephyrproject/zephyr/include/kernel_includes.h:33,
                 from /Users/joshfischer/zephyrproject/zephyr/include/kernel.h:17,
                 from /Users/joshfischer/zephyrproject/zephyr/include/zephyr.h:18,
                 from /Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c:8:
/Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c: In function 'main':
/Users/joshfischer/zephyrproject/zephyr/include/devicetree.h:213:32: error: 'DT_N_ALIAS_i2c_0_P_label' undeclared (first use in this function)
  213 | #define DT_ALIAS(alias) DT_CAT(DT_N_ALIAS_, alias)
      |                                ^~~~~~~~~~~
/Users/joshfischer/zephyrproject/zephyr/include/devicetree.h:2542:24: note: in definition of macro 'DT_CAT'
 2542 | #define DT_CAT(a1, a2) a1 ## a2
      |                        ^~
/Users/joshfischer/zephyrproject/zephyr/include/devicetree.h:652:27: note: in expansion of macro 'DT_PROP'
  652 | #define DT_LABEL(node_id) DT_PROP(node_id, label)
      |                           ^~~~~~~
/Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c:13:17: note: in expansion of macro 'DT_LABEL'
   13 | #define I2C_DEV DT_LABEL(DT_ALIAS(i2c_0))
      |                 ^~~~~~~~
/Users/joshfischer/zephyrproject/zephyr/include/devicetree.h:213:25: note: in expansion of macro 'DT_CAT'
  213 | #define DT_ALIAS(alias) DT_CAT(DT_N_ALIAS_, alias)
      |                         ^~~~~~
/Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c:13:26: note: in expansion of macro 'DT_ALIAS'
   13 | #define I2C_DEV DT_LABEL(DT_ALIAS(i2c_0))
      |                          ^~~~~~~~
/Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c:80:31: note: in expansion of macro 'I2C_DEV'
   80 |  i2c_dev = device_get_binding(I2C_DEV);
      |                               ^~~~~~~
/Users/joshfischer/zephyrproject/zephyr/include/devicetree.h:213:32: note: each undeclared identifier is reported only once for each function it appears in
  213 | #define DT_ALIAS(alias) DT_CAT(DT_N_ALIAS_, alias)
      |                                ^~~~~~~~~~~
/Users/joshfischer/zephyrproject/zephyr/include/devicetree.h:2542:24: note: in definition of macro 'DT_CAT'
 2542 | #define DT_CAT(a1, a2) a1 ## a2
      |                        ^~
/Users/joshfischer/zephyrproject/zephyr/include/devicetree.h:652:27: note: in expansion of macro 'DT_PROP'
  652 | #define DT_LABEL(node_id) DT_PROP(node_id, label)
      |                           ^~~~~~~
/Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c:13:17: note: in expansion of macro 'DT_LABEL'
   13 | #define I2C_DEV DT_LABEL(DT_ALIAS(i2c_0))
      |                 ^~~~~~~~
/Users/joshfischer/zephyrproject/zephyr/include/devicetree.h:213:25: note: in expansion of macro 'DT_CAT'
  213 | #define DT_ALIAS(alias) DT_CAT(DT_N_ALIAS_, alias)
      |                         ^~~~~~
/Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c:13:26: note: in expansion of macro 'DT_ALIAS'
   13 | #define I2C_DEV DT_LABEL(DT_ALIAS(i2c_0))
      |                          ^~~~~~~~
/Users/joshfischer/zephyrproject/zephyr/samples/drivers/i2c_fujitsu_fram/src/main.c:80:31: note: in expansion of macro 'I2C_DEV'
   80 |  i2c_dev = device_get_binding(I2C_DEV);
      |                               ^~~~~~~
[21/100] Building C object zephyr/arch/arch/arm/core/aarch32/mpu/CMakeFiles/arch__arm__core__aarch32__mpu.dir/arm_mpu.c.obj
ninja: build stopped: subcommand failed.
FATAL ERROR: command exited with status 1: /usr/local/bin/cmake --build /Users/joshfischer/zephyrproject/build
```

**Environment (please complete the following information):**
 - OS: (MacOS BigSur 11.1 - M1)
 - Toolchain: Zephyr SDK
 - Version: 2.6.0


**Additional context**
Other examples will build successfully for me.  Not sure why this one will not.

--- a/samples/drivers/i2c_fujitsu_fram/sample.yaml
+++ b/samples/drivers/i2c_fujitsu_fram/sample.yaml
@@ -4,7 +4,7 @@ tests:
   sample.drivers.i2c.fujitsu_fram:
     tags: drivers
     depends_on: i2c
-    filter: dt_alias_exists("i2c-0")
+    filter: dt_nodelabel_enabled("i2c0")
     harness: console
     harness_config:
         type: one_line

--- a/samples/drivers/i2c_fujitsu_fram/src/main.c
+++ b/samples/drivers/i2c_fujitsu_fram/src/main.c
@@ -10,10 +10,8 @@
 #include <device.h>
 #include <drivers/i2c.h>
 
-#define I2C_NODE DT_ALIAS(i2c_0)
-
 /**
- * @file Sample app using the Fujitsu MB85RC256V FRAM through ARC I2C.
+ * @file Sample app using the Fujitsu MB85RC256V FRAM through I2C.
  */
 
 #define FRAM_I2C_ADDR	0x50
@@ -72,7 +70,7 @@ static int read_bytes(const struct device *i2c_dev, uint16_t addr,
 
 void main(void)
 {
-	const struct device *i2c_dev = DEVICE_DT_GET(I2C_NODE);
+	const struct device *i2c_dev = DEVICE_DT_GET(DT_NODELABEL(i2c0));
 	uint8_t cmp_data[16];
 	uint8_t data[16];
 	int i, ret;

